generators/database-changelog-liquibase/templates/src/main/resources/config/liquibase/fake-data/table_entity.csv.ejs

```diff
@@ -59,27 +52,21 @@ for (lineNb = 0; lineNb < entity.fakeDataCount; lineNb++) {
     }
 
     line.push(data);
-    if (field.columnType === 'blob' || field.columnType === 'longblob') {
+      if (field.fieldType === 'byte[]' && field.fieldTypeBlobContent !== 'text') {
```

Comment:
There are also derived properties for fieldTypeBytes and fieldTypeBlobContentText